### PR TITLE
[Wallet] Fix premature hiding of syncing banner and increase some timeouts

### LINKED ISSUE #1775: App doesn't seem to be ultralight syncing
### Expected Behavior

Syncs a small amount of recent blocks

### Current Behavior

Web3 seems to wait for thousands of blocks, but this still happens relatively quickly. Too quickly to be actually syncing I suspect.

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

The app was prematurely hiding the Connecting... banner because it assumed that once geth received some blocks, the app was already synced. In fact, we must wait for web3 to be fully synced.

### Tested

Syncing after fresh install on android and ios, both default and zero-sync mode
Confirmed invite redemption properly waits for syncing to be done

### Other changes

Wait for sync to be done before redeeming invite and don't include that time in the redeem invite 
timeout

Fix missing k8s cluster check in celotool invite script

Increase some timeouts

### Related issues

- Fixes #1775

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/celotool/src/cmds/account/invite.ts [hunk 1]
```diff
@@ -25,6 +26,7 @@ export const builder = (yargs: Argv) => {
 }
 
 export const handler = async (argv: InviteArgv) => {
+  await switchToClusterFromEnv()
```
Comment (jeanregisser) [inline packages/celotool/src/cmds/account/invite.ts:29]: Nice! 👍

File: packages/mobile/src/geth/geth.ts [hunk 2]
```diff
@@ -79,7 +79,7 @@ async function createNewGeth(): Promise<typeof RNGeth> {
   const genesis: string = await readGenesisBlockFile(nodeDir)
   const networkID: number = GenesisBlockUtils.getChainIdFromGenesis(genesis)
 
-  Logger.debug('Geth@newGeth', `Network ID is ${networkID}`)
+  Logger.debug('Geth@newGeth', `Network ID is ${networkID}, syncMode is ${syncMode}`)
```
Comment (jeanregisser) [inline packages/mobile/src/geth/geth.ts:82]: 👍

File: packages/mobile/src/invite/saga.ts [hunk 3]
```diff
@@ -46,7 +46,7 @@ import { zeroSyncSelector } from 'src/web3/selectors'
 
 const TAG = 'invite/saga'
 export const TEMP_PW = 'ce10'
-export const REDEEM_INVITE_TIMEOUT = 1 * 60 * 1000 // 1 minute
+export const REDEEM_INVITE_TIMEOUT = 2 * 60 * 1000 // 2 minutes
```
Comment (jeanregisser) [inline packages/mobile/src/invite/saga.ts:49]: 1 min feels already quite long. No way around this?

File: packages/mobile/src/web3/reducer.ts [hunk 4]
```diff
@@ -71,9 +67,14 @@ export const reducer = (
...
-    case Actions.SET_BLOCK_NUMBER:
+    case Actions.COMPLETE_WEB3_SYNC:
       return {
         ...state,
+        syncProgress: {
+          startingBlock: state.syncProgress.startingBlock,
+          currentBlock: action.latestBlockNumber,
+          highestBlock: action.latestBlockNumber,
+        },
         latestBlockNumber: action.latestBlockNumber,
```
Comment (jeanregisser) [inline packages/mobile/src/web3/reducer.ts:78]: Is `latestBlockNumber` still needed?

File: packages/mobile/src/web3/saga.ts [hunk 5]
```diff
@@ -48,40 +49,45 @@ const MNEMONIC_BIT_LENGTH = (ETH_PRIVATE_KEY_LENGTH * 8) / 2
...
 const BLOCK_CHAIN_CORRUPTION_ERROR = "Error: CONNECTION ERROR: Couldn't connect to node on IPC."
 
 // checks if web3 claims it is currently syncing and attempts to wait for it to complete
 export function* checkWeb3SyncProgress() {
-  const zeroSyncMode = yield select(zeroSyncSelector)
-  if (zeroSyncMode) {
-    // In this mode, the check seems to fail with
-    // web3/saga/checking web3 sync progress: Error: Invalid JSON RPC response: "":
-    return true
-  }
```
Comment (jeanregisser) [inline packages/mobile/src/web3/saga.ts:61]: We want to run the loop now in zeroSyncMode too?

**Review by jeanregisser — APPROVED** [review]
Great! Thanks for digging this! 💯 👍 
Left a few questions.

File: packages/mobile/src/invite/saga.ts (hunk 3 above)
Comment (jmrossy) [reply packages/mobile/src/invite/saga.ts:49]: Until we improve the speed of account creation and txs on low end phones, 1 min is just too short for all the things we do here

File: packages/mobile/src/web3/reducer.ts (hunk 4 above)
Comment (jmrossy) [reply packages/mobile/src/web3/reducer.ts:78]: It's used in one place. Not strictly need but I have a feeling we may need it eventually so didn't remove it

File: packages/mobile/src/web3/saga.ts (hunk 5 above)
Comment (jmrossy) [reply packages/mobile/src/web3/saga.ts:61]: Yes

**Pull request merged**